cxx-qt-build/src/lib.rs

```diff
@@ -9,6 +9,7 @@ use serde::{Deserialize, Serialize};
 use std::env;
 use std::fs::File;
 use std::io::Write;
+use std::path::PathBuf;
```

Comment:
Can we split this first commit of using `PathBuf` into a separate pull request, as we can likely land this first and faster, therefore reducing noise in this second CLI pull request?

Reply:
I literally meant take out the first of three commits in this branch https://github.com/KDAB/cxx-qt/pull/161/commits/9fbd45f67f88f082098334c0900bec0ffb10f0e3 and submit by itself.